tests/test_grades/test_csv.py

```diff
@@ -413,8 +413,7 @@ def test_import_success_no_header(self):
 
     def test_import_file_error(self):
         expected_file_error_msg = (
-            "Error: line contains NUL. Are you sure the file is a "
-            "CSV file other than a Microsoft Excel file?")
+            "Error: line contains NUL")
```

Comment:
If you remove this change from your branch, your CI should pass.

Reply:
Yeah, I ended up with a three-way merge on that block and bet on the wrong horse.  I daresay everything passes now.

BUT it's going to be pulling in the default Docker image, not the new one.  See `docker_image_run_py/Dockerfile` for details.  We should definitely test that before a merge.  It works locally but that worth about the letters it takes to type it.